lib/iex/lib/iex/autocomplete.ex

```diff
@@ -295,29 +309,35 @@ defmodule IEx.Autocomplete do
   end
 
   defp match_module_funs(funs, hint) do
-    for({fun, arity} <- funs,
-        name = Atom.to_string(fun),
-        String.starts_with?(name, hint),
-        do: %{kind: :function, name: name, arity: arity})
+    for({fun, arity} <- funs, name = Atom.to_string(fun), String.starts_with?(name, hint), do: %{
```

Comment:
If you remove the parenthesis or change to proper blocks this `for` and the one below should format better.